electron_app/AuthLoginModule.js

```diff
@@ -0,0 +1,99 @@
+const electronOauth2 = require('electron-oauth2');
+const constants = require('./helpers/enviroment').constants;
+
+class LoginGoogle {
+  static setWindows(win) {
+    return {
+      parent: win,
+      modal: true,
+      frame: false,
+      resizable: false,
+      alwaysOnTop: true,
+      autoHideMenuBar: true,
+      webPreferences: {
+        nodeIntegration: false
+      }
+    };
+  }
+  constructor(googleOptions, googleConfig, win, app, status) {
+    this.googleOptions = googleOptions || null;
+    this.googleConfig = googleConfig || null;
+    this.myApiOauth = electronOauth2(this.googleConfig, LoginGoogle.setWindows(win)) || null;
+    this.win = win || null;
+    this.app = app || null;
+    this.status = status || false;
```

Comment:
Meaning of `status` is unclear in this code without a comment. How about changing it to `isLoggedIn`?